explainer.md

```diff
@@ -52,17 +53,22 @@ goals of the project are, and a brief overview of how the solution works.
 This should be no more than 1-2 paragraphs.]
 
 
-## Goals [or Motivating use cases, or Scenarios]
-
-[What is the end-user need which this project aims to address?]
-
+## Goals
+- Allow web applications to evict partitions of data
+- Allow web developers to specify eviction ordering
+- Allow web developers to remove a service workers without clearing data for the entire origin
```

Comment:
I don't understand this one?

Reply:
I was trying to reword the use case for associating service worker with bucket to make entire service worker footprint eviction easy. I'll try re-phrasing a bit.